cli/src/complete_job.rs

```diff
@@ -0,0 +1,37 @@
+use crate::{Debug, ExecuteZeebeCommand};
+use async_trait::async_trait;
+use clap::Args;
+use color_eyre::eyre::Result;
+use zeebe_client::{
+    api::{CompleteJobRequest, CompleteJobResponse},
+    ZeebeClient,
+};
+
+#[derive(Args, Clone, Debug)]
+pub(crate) struct CompleteJobArgs {
+    job_key: i64,
+
+    #[clap(long, required = false, default_value = "")]
```

Comment:
:thought_balloon: Is there a way to give the user a hint that this is supposed to be a comma separated list?

If not, we probably should document this.